source/fenwick.h

```diff
@@ -25,4 +25,22 @@ struct Fenwick {
 	Value sum(int a, int b) { return sum(b) - (a?sum(a-1):0); }
 	Value get(int p) { return sum(p, p); }
 	void set(int p, Value v) { add(p, v - get(p)); }
+
+	/* Optional. If the fenwick has no negative values returns the smallest i
+		for which sum(0, i) >= k in O(log N) (or -1 if such i does not exist)
+	*/
```

Comment:
```suggestion
	// Optional. If the fenwick has no negative values returns the smallest i
	// for which sum(0, i) >= k in O(log N) (or -1 if such i does not exist)
```
Nitpick, this would save a line